utils/scripts/dbaas_utils.py

```diff
@@ -372,3 +372,35 @@ def create_provision_instance_yaml(isv, prov_acc_ns, deploy_instance_ns):
         data["spec"]["otherInstanceParams"]["projectName"] = instance
     print(data)
     return yaml.dump(data, sort_keys=False)
+
+
+def add_policy_connection_namespaces(namespace: list):
+    """Adding Connection Namespace on DBaaSPolicy"""
+    sl = BuiltIn().get_library_instance("SeleniumLibrary")
+    btn_add_conn_xpath = "//button[@id='root_spec_connectionNamespaces_add-btn']"
+    if not namespace:
+        namespace = "*"
+    for i, ns in enumerate(namespace):
+        sl.click_element(btn_add_conn_xpath)
+        txtbx_value_xpath = "//input[@id='root_spec_connectionNamespaces_" + str(i) + "']"
```

Comment:
Same here, we need to move it to  resources/object_repo/create_dbaaspolicy_obj.resource.